redpanda/templates/post-upgrade-configmap.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "redpanda.fullname" . }}-post-upgrade
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+{{ include "common-labels" . | nindent 4 }}
+data:
+  post-upgrade.sh: |-
+    {{ .Files.Get "post-upgrade.sh" | nindent 4 }}
```

Comment:
Why use an extra configmap for this instead of using the file as the args in the job?

Reply:
I find this way of doing it much more readable and there are the added benefits of normal editor niceties if/when it needs to be touched and easier addition of temporary debugging changes should they be necessary. Never much liked showing shell into yaml.

Reply:
I do really like using .File for this, but wouldn't it be just as readable to:
```
kind: Job
spec:
  template:
    spec:
      containers:
        - command: ["/bin/sh", "-c"]
          args:
            - >
              {{ .Files.Get "post-upgrade.sh" | nindent 14 }}
```

Reply:
Honestly, that never even crossed my mind! I'm blaming too much time in Kustomize, although that brings up the point that it's easier to patch a configmap should someone want to.